gradle/dependencies.gradle

```diff
@@ -44,15 +45,14 @@ ext {
                   "org.springframework:spring-web:$spring",
                   "org.springframework:spring-webmvc:$spring",
                   "org.springframework:spring-context:$spring",
+                  "org.springframework.boot:spring-boot-autoconfigure:$springBoot"
           ],
           clientProvided: [
                   "javax.servlet:javax.servlet-api:$servlet",
                   "com.fasterxml.jackson.core:jackson-core:${jackson}",
                   "com.fasterxml.jackson.core:jackson-databind:${jackson}",
-                  "joda-time:joda-time:$joda"
-          ],
-          client        : [
                   "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${jackson}",
+                  "joda-time:joda-time:$joda",
```

Comment:
I kept it as an optional dependency, I removed the client part and added the yaml as another optional dependency. it shows this line as green because I added the yaml dependency before it.